Makefile.build

```diff
@@ -362,9 +362,14 @@ else
   DYNDEP=-dyndep opt
 endif
 
+# can easily produce too long command line with many .v files prefixed by build_vo
+# so we use find + xargs to split coqdep invocation if necessary
+# OSX sed doesn't support -null-data so can't use -print0
 $(VFILED): $(BUILD_VFILES) $(BUILD_MLFILES) $(COQDEP) | all-src
 	$(SHOW)'COQDEP    VFILES'
-	$(HIDE)$(COQDEP) -boot $(DYNDEP) -R $(VO_OUT_DIR)theories Coq -Q $(VO_OUT_DIR)user-contrib "" $(PLUGININCLUDES) $(USERCONTRIBINCLUDES) $(BUILD_VFILES) $(TOTARGET)
+	$(HIDE)find theories $(addprefix user-contrib/, $(USERCONTRIBDIRS)) -type f -name '*.v' | \
+	sed 's|^|$(VO_OUT_DIR)|' | \
```

Comment:
Why BUILD_VFILES is not used? Why is `sed` needed? Both look fishy to me.

Reply:
because it's too long

Reply:
I mean the find should not be there, otherwise we cannot keep things in sync.

Reply:
The find is the whole point, we cannot use makefile variables as they are too long.